src/config.inc

```diff
@@ -76,7 +79,7 @@ endif
 OSX_IDENTITY="Developer ID Application: Daniel Kroening"
 
 # Detailed version information
-CBMC_VERSION = 5.26.0
+CBMC_VERSION = 5.26.1
```

Comment:
I think you may have rebased on to a branch other than `develop`?

Reply:
On top of the version PR #6014